src/main/java/com/iland/coda/footprint/GenerationDate.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022, iland Internet Solutions, Corp
+ *
+ * This software is licensed under the Terms and Conditions contained within the
+ * "LICENSE.txt" file that accompanied this software. Any inquiries concerning
+ * the scope or enforceability of the license should be addressed to:
+ *
+ * iland Internet Solutions, Corp
+ * 1235 North Loop West, Suite 800
+ * Houston, TX 77008
+ * USA
+ *
+ * http://www.iland.com
+ */
+
+package com.iland.coda.footprint;
+
+import static java.time.ZoneOffset.UTC;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+/**
+ * {@link GenerationDate}.
+ *
+ * @author <a href="mailto:tagspilman@1111systems.com">Tag Spilman</a>
+ */
+public final class GenerationDate {
+
```

Comment:
Shared static field could cause issues when multiple threads. 
See [this](https://www.java67.com/2013/01/how-to-format-date-in-java-simpledateformat-example.html#:~:text=In%20order%20to%20format%20dates,M%20means%20Month%20of%20year.)

Reply:
O.o I had no idea. Nice catch!